### ISSUE TITLE
Remove\replace file from drive by file name not by ID

### ISSUE THREAD

**Issue by alexkmbk** [prasmussen/gdrive#364]
I would like to remove or replace file by file name not by ID. Don't see any special option for that.

**Comment by DRAKUN** [comment]
That is good suggestion because we have limited space in our drive

**Comment by pipilupipixia** [comment]
But Google Drive allows you put lots of file with the same name in one directory. You cant tell the difference between them if you only give a file name.

**Comment by alexkmbk** [comment]
> But Google Drive allows you put lots of file with the same name in one directory. You cant tell the difference between them if you only give a file name.

Yes, that means I want to remove them all. The use case: remove backup files to upload the new ones without using extra space on gdrive.

**Comment by kkrylov** [comment]
```
/root/gdrive list | grep site.ru.sql.gz | awk '{print $1}' | xargs -I {} sh -c '/root/gdrive delete {}'
/root/gdrive upload /var/www/backup/site.ru.sql.gz
```
